app/components/curriculum-inventory-report-list-item.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import ResolveAsyncValue from 'ilios-common/classes/resolve-async-value';
+import PermissionChecker from 'ilios/classes/permission-checker';
+import { use } from 'ember-could-get-used-to-this';
+import { inject as service } from '@ember/service';
+
+export default class CurriculumInventoryReportListItemComponent extends Component {
+  @service iliosConfig;
+  @service permissionChecker;
+
+  @tracked userCanDelete = false;
+  @tracked showConfirmRemoval;
+
+  @use canDeletePermission = new PermissionChecker(() => ["canDeleteCurriculumInventoryReport", this.args.report]);
+  @use academicYearCrossesCalendarYearBoundaries = new ResolveAsyncValue(() => [
+    this.iliosConfig.itemFromConfig('academicYearCrossesCalendarYearBoundaries')
+  ]);
+
+  get canDelete() {
```

Comment:
Since this just echos the Resource I think `canDeletePermission` can just be changed to `canDelete` making this getter unnecessary.

Reply:
:heavy_check_mark: